File: src/main/java/com/projectswg/holocore/resources/support/global/commands/callbacks/chat/CmdBroadcastArea.java
package com.projectswg.holocore.resources.support.global.commands.callbacks.chat;

import com.projectswg.holocore.intents.support.global.chat.SystemMessageIntent;
import com.projectswg.holocore.resources.support.global.commands.ICmdCallback;
import com.projectswg.holocore.resources.support.global.player.Player;
import com.projectswg.holocore.resources.support.objects.swg.SWGObject;
import org.jetbrains.annotations.NotNull;

public final class CmdBroadcastArea implements ICmdCallback {
	
	@Override
	public void execute(@NotNull Player player, SWGObject target, @NotNull String args) {
		SystemMessageIntent.broadcastArea(player, args);
	}
	
}

File: src/main/java/com/projectswg/holocore/resources/support/global/commands/callbacks/chat/CmdBroadcastArea.kt
package com.projectswg.holocore.resources.support.global.commands.callbacks.chat

import com.projectswg.holocore.intents.support.global.chat.SystemMessageIntent.Companion.broadcastArea
import com.projectswg.holocore.resources.support.global.commands.ICmdCallback
import com.projectswg.holocore.resources.support.global.player.Player
import com.projectswg.holocore.resources.support.objects.swg.SWGObject

class CmdBroadcastArea : ICmdCallback {
	override fun execute(player: Player, target: SWGObject?, args: String) {
		broadcastArea(player, args)
	}
}
